app/webutil.js

```diff
@@ -278,4 +278,73 @@ WebUtil.injectParamIfMissing = function (path, param, value) {
     }
 };
 
+// Dynamically load scripts without using document.write()
+// Reference: http://unixpapa.com/js/dyna.html
+//
+// Handles the case where load_scripts is invoked from a script that
+// itself is loaded via load_scripts. Once all scripts are loaded the
+// window.onscriptsloaded handler is called (if set).
+WebUtil.get_include_uri = function (root_dir) {
+    return (typeof INCLUDE_URI !== "undefined") ? INCLUDE_URI + root_dir + '/' : root_dir + '/';
+};
+WebUtil._loading_scripts = [];
+WebUtil._pending_scripts = [];
+WebUtil.load_scripts = function (files_by_dir) {
+    "use strict";
+    var head = document.getElementsByTagName('head')[0], script,
+        ls = WebUtil._loading_scripts, ps = WebUtil._pending_scripts;
+
+    var loadFunc = function (e) {
+        while (ls.length > 0 && (ls[0].readyState === 'loaded' ||
+                                 ls[0].readyState === 'complete')) {
+            // For IE, append the script to trigger execution
+            var s = ls.shift();
+            //console.log("loaded script: " + s.src);
+            head.appendChild(s);
+        }
+        if (!this.readyState ||
+            (Util.Engine.presto && this.readyState === 'loaded') ||
+            this.readyState === 'complete') {
+            if (ps.indexOf(this) >= 0) {
+                this.onload = this.onreadystatechange = null;
+                //console.log("completed script: " + this.src);
+                ps.splice(ps.indexOf(this), 1);
+
+                // Call window.onscriptsload after last script loads
+                if (ps.length === 0 && window.onscriptsload) {
+                    window.onscriptsload();
+                }
+            }
+        }
+    };
+
+    var root_dirs = Object.Keys(files_by_dir);
```

Comment:
fixed.  I swear I fixed that one, must have gotten mangled when I went to stick it in the right commit.